.github/actions/core-tests/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Core tests'
+description: 'Run core and entrypoint tests'
+runs:
+  using: 'composite'
+  steps:
+    - name: Core tests
+      if: always()
+      run: npm run test:core -- --reporter json --reporter-option 'output=reports/core.json'
+      shell: bash
+
+    - name: Entrypoint tests
+      if: always()
+      run: npm run test:entrypoint -- --reporter json --reporter-option 'output=reports/entrypoint.json'
```

Comment:
I can't really remember why `:entrypoint` (which is only one test) needs to be a different test suite than `:core`. I think I'm going to leave them how they are for now and open that worm-can another day. It is not a huge deal.